public/503.php

```diff
@@ -0,0 +1,69 @@
+<?php
+header('HTTP/1.1 503 Service Temporarily Unavailable');
+header('Status: 503 Service Temporarily Unavailable');
+?><!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>503 | Service Unavailable</title>
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet">
+
+    <style>
+        body {
+            background-color: #F5F5F5;
+            font-family: Arial, sans-serif;
+            display: flex;
+            flex-direction: column;
+            min-height: 100vh;
+        }
+
+        .container {
+            text-align: center;
+            margin-top: 100px;
+            flex: 1;
+        }
+
+        .header-image {
+            max-width: 100%;
+            height: auto;
+            margin-bottom: 30px;
+        }
+
+        .message-title {
+            font-size: 48px;
+            font-weight: bold;
+            margin-bottom: 20px;
+        }
+
+        .message-content {
+            font-size: 24px;
+            margin-bottom: 50px;
+        }
+
+        .footer {
+            background-color: #009FDE;
+            color: white;
+            padding: 15px;
+            text-align: center;
+        }
+    </style>
+</head>
+<body>
+<div class="container">
+    <img src="/build/assets/Hero-DagNCfZ2.svg" alt="Maintenance Image" class="header-image">
```

Comment:
I don't see this asset `Hero-DagNCfZ2.svg`

Reply:
It already exists, I think it gets deployed there with NPM.
I just copied the path from the homepage.

Reply:
Ah it is `Hero.svg`